build.zig

```diff
@@ -178,6 +196,11 @@ pub fn build_blob(b: *Builder, arch: builtin.Arch, target_name: []const u8, path
   return blob(b, elf, .Padded);
 }
 
+pub fn build_pi3_kernel8(b: *Builder, arch: builtin.Arch, target_name: []const u8, path_prefix: []const u8) !*TransformFileCommandStep {
+  const elf = try build_elf(b, arch, target_name, path_prefix);
+  return pi3_kernel8(b, elf);
```

Comment:
Just call `blob(b, elf, .NotPadded)` and scrap `pi3_kernel8`? Or am I missing something?